src/tests/unit/tests/DetailsView/handlers/assessment-instance-table-handler.test.tsx

```diff
@@ -220,6 +283,11 @@ describe('AssessmentInstanceTableHandlerTest', () => {
         testSubject.addPathForValidation(path);
     });
 
+    test('clearPathSnippetData', () => {
+        actionMessageCreatorMock.setup(a => a.clearPathSnippetData()).verifiable(Times.once());
```

Comment:
If you remove the call to clearPathSnippetData in your code, does this test fail? It seems like we'd need a `verifyAll` somewhere but I don't see one

Reply:
Great point--just changed these for my two added tests. The concerning thing is the rest of the functions don't include a call to verifyAll (like the addFailureInstance test right below mine) so they're all passing without verification too... Should I try and refactor the entire file?

Reply:
I would check that we need this change, either by forcing any of these tests to fail & seeing if they still pass (like the earlier suggestion), or seeing if an absurd expectation (`Times.exactly(49)` instead of `Times.once()`) still passes. If so, it might be doable to just add `actionMessageCreatorMock.verifyAll()` in each test that has a `setup` for it (I count 5, 3 of which are about path/snippets)

Reply:
Cool--I checked the previous suggestion of taking out the call to the function on the testSubject and they failed with the verifyAll() included. I'll add them in!

Reply:
I've found 5 that I've added to (addPathForValidation, clearPathSnippetData, addFailureInstance, passUnmarkedInstance, and updateFocusedInstance)